mandatory/2-function-creation.js

```diff
@@ -15,27 +23,64 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice >= 200) {
+    return totalPrice * 0.9;
+  }
+  else {
+    return totalPrice * 0.95;
 
-/*
+  }
+
+
+
+}
+
+
+/*let limit = 0;
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+
+  for (let i = 0; i <= limit; i++) {
+    if (i % 2 === 1) {
+      console.log(i);
+    }
+  }
+
+
+}
```

Comment:
Ensure you do not leave lots of unnecessary whitespace in functions.

Reply:
Well done finding an alternative solution, but also ensure you read the requirements carefully, it asks for a while loop to be used.